src/libs/IOUUtils.js

```diff
@@ -0,0 +1,63 @@
+import CONST from '../CONST';
+
+/**
+ * Calculates the amount per user given a list of participants
+ * @param {Array} participants
+ * @param {Number} total
+ * @param {Boolean} isDefaultUser
+ * @returns {Number}
+ */
+function calculateAmount(participants, total, isDefaultUser = false) {
```

Comment:
I'm guessing that the `participants` don't include the owner of the IOU, is this correct? if it is, maybe we should mention it.